### PR TITLE
snaps-skunkworks@0.10.2

### PR DESCRIPTION
**Pull request by rekmarks**
Updates Snaps skunkworks packages to `0.10.2`.

### CODE REVIEW

File: app/scripts/metamask-controller.js [hunk 1]
```diff
@@ -580,7 +580,8 @@ export default class MetamaskController extends EventEmitter {
     this.workerController = new IframeExecutionService({
       onError: this.onExecutionEnvironmentError.bind(this),
       iframeUrl: new URL(
-        'https://metamask.github.io/iframe-execution-environment/0.3.1',
+        // 'https://metamask.github.io/iframe-execution-environment/0.3.1',
+        'http://localhost:6363',
```
Comment (FrederikBolding) [inline app/scripts/metamask-controller.js:584]: Remember to revert and update to the new release 😄

File: patches/@metamask+snap-controllers+0.10.0.patch [hunk 2]
```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/@metamask/snap-controllers/dist/snaps/SnapController.js b/node_modules/@metamask/snap-controllers/dist/snaps/SnapController.js
+index 72d00f7..0dd8f30 100644
+--- a/node_modules/@metamask/snap-controllers/dist/snaps/SnapController.js
++++ b/node_modules/@metamask/snap-controllers/dist/snaps/SnapController.js
+@@ -810,6 +810,7 @@ class SnapController extends controllers_1.BaseControllerV2 {
+                     reject(new Error('The request timed out.'));
+                 }, this._maxRequestTime);
+             });
++            request.jsonrpc = '2.0';
```
Comment (FrederikBolding) [inline patches/@metamask+snap-controllers+0.10.0.patch:9]: Should be reverted as part of `0.10.1`

Comment (FrederikBolding) [inline patches/@metamask+snap-controllers+0.10.0.patch:9]: Should be reverted as part of `0.10.1`

File: shared/constants/permissions.js [hunk 3]
```diff
@@ -21,4 +21,7 @@ export const PermissionNamespaces = Object.freeze({
 export const EndowmentPermissions = Object.freeze({
   'endowment:network-access': 'endowment:network-access',
 });
+
+// Methods / permissions in external packages that we are temporarily excluding.
+export const ExcludedSnapPermissions = new Set(['snap_notify']);
```
Comment (FrederikBolding) [inline shared/constants/permissions.js:26]: Not for long 😉

Comment (FrederikBolding) [inline shared/constants/permissions.js:26]: Not for long 😉

File: lavamoat/browserify/main/policy.json [hunk 4]
```diff
@@ -695,6 +695,7 @@
     },
```
Comment (FrederikBolding) [inline lavamoat/browserify/main/policy.json:695]: I'm not a 100% sure how to review these, but it looks like changes match up with changes made to `skunkworks` 👍

[the remark this answers is not in the document]
Comment (rekmarks) [reply lavamoat/browserify/main/policy.json:695]: I think the most important thing is to call out any strange additions.

**Review by FrederikBolding — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**